tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -192,8 +192,8 @@ function sortTableDataItems(
     let bValue = b[sort.name];
 
     if (sort.name === 'experimentAlias') {
-      aValue = (aValue as ExperimentAlias).aliasNumber;
-      bValue = (bValue as ExperimentAlias).aliasNumber;
+      aValue = a.experimentAlias!.aliasNumber;
```

Comment:
This should always be set in the runs table so the non null assertion should be fine. I don't mind adding a null check to the if statement on line 194 though 🤷.

Reply:
This is now irrelevant.